internal/receiver/databricksreceiver/README.md

```diff
@@ -1,20 +1,25 @@
-# Databricks Receiver (Alpha)
+# Databricks Receiver
 
 The Databricks Receiver uses the Databricks
 [API](https://docs.databricks.com/dev-tools/api/latest/index.html)
 to generate metrics about the operation of a Databricks instance.
 
+In addition to generating metrics from the Databricks API, it also generates metrics from the Spark subsystem running in a Databricks instance.
+
 Supported pipeline types: `metrics`
 
-> :construction: This receiver is in **ALPHA**. Behavior, configuration fields, and metric data model are subject to change.
+> :construction: This receiver is in **DEVELOPMENT**. Behavior, configuration fields, and metric data model are subject to change.
 
 ## Configuration
 
 The following fields are required:
 
 - `instance_name`: A string representing the name of the instance. This value gets set as a `databricks.instance.name` resource attribute.
-- `endpoint`: The protocol (http or https), hostname, and port for the Databricks API, without a trailing slash.
+- `endpoint`: The URL containing a protocol (http or https), hostname, and (optional) port of the Databricks API, without a trailing slash.
 - `token`: An [access token](https://docs.databricks.com/dev-tools/api/latest/authentication.html) to authenticate to the Databricks API. 
+- `spark_org_id`: The Spark Org ID. See the Spark Subsystem Properties section below for how to get this value.
+- `spark_api_url`: The URL containing a protocol (http or https), hostname, and (optional) port of the Spark API. See the Spark Subsystem Properties section below for how to get this value.
```

Comment:
Since "endpoint" is used conventionally elsewhere would `spark_endpoint` make sense?

Reply:
Yes. I had the same idea and made the change but apparently it somehow got lost. Fixed.